dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -407,12 +362,12 @@ def test_wrong_parameter_error(Cluster):
         re.DOTALL,
     )
     with pytest.raises(ValueError, match=match):
-        create_cluster_func(
-            Cluster, cores=1, memory="1GB", wrong_parameter="wrong_parameter_value"
-        )
+        Cluster(cores=1, memory="1GB", wrong_parameter="wrong_parameter_value")
 
 
-def test_security():
+@pytest.mark.xfail_ci({"htcondor": "#535 no shared filesystem in htcondor ci"})
+@pytest.mark.xfail_ci({"slurm": "#535 no shared filesystem in slurm ci"})
+def test_security(EnvSpecificCluster, loop):
```

Comment:
This was only tested with LocalCluster before, so that it would not fail although a shared file system is really needed for this to work.